templates/receipt/README.md

```diff
@@ -0,0 +1,6 @@
+Sample receipt document
+======================
+This example shows creating a sample receipt document using the creator templates.  
```

Comment:
Please drop the `the` from this sentence:
```
This example shows creating a sample receipt document using creator templates.
```
Or just use the comment in the example main file:
```
This example showcases the usage of creator templates to create a receipt document.
```